config/prod/ecmonsen-emorypipeline-lambda.yaml

```diff
@@ -0,0 +1,22 @@
+template_path: "remote/ecmonsen-emorypipeline-lambda.yaml"
+stack_name: "ecmonsen-emorypipeline-lambda"
+dependencies:
+  - prod/ecmonsen-emorypipeline-ci.yaml
+parameters:
+  SourceAccountId: "186706593821"
```

Comment:
i'm guessing your developing in the `develop` branch of ecmonsen-emorypipeline-lambda-code repo ?  it would not work with the template in `master` branch.

Reply:
After our Slack conversation, I changed ecmonsen-emorypipeline-lambda-code to only have a master branch, where all development will take place going forward.